### PR TITLE
define hnsec

### PR DESCRIPTION
**Pull request by WalterBright**
Define it instead of wondering what it means.

### CODE REVIEW

File: std/datetime/systime.d [hunk 1]
```diff
@@ -446,6 +446,8 @@ private:
     calendar-based operations, and getting individual units from it such as
     years or days is going to involve conversions and be less efficient.
 
+    An $(I hnsec) is 100 nanoseconds. There are 10,000,000 hnsecs in a second.
```
Comment (ljmf00) [inline std/datetime/systime.d:449]:
```suggestion
    An $(I hnsec) (hecto-nanosecond) is 100 nanoseconds. There are 10,000,000 hnsecs in a second.
```

I would add the extended name as there are also references to that version.

File: std/datetime/systime.d [hunk 2]
```diff
@@ -446,6 +446,8 @@ private:
     calendar-based operations, and getting individual units from it such as
     years or days is going to involve conversions and be less efficient.
 
+    An $(I hnsec) (hecto-nanosecond) is 100 nanoseconds. There are 10,000,000 hnsecs in a second.
+
```
Comment (Geod24) [inline std/datetime/systime.d:450]: https://github.com/dlang/druntime/blob/5a77d7fc0b3d3e1857a91915081ea26aa568de3c/src/core/time.d#L8-L14

Comment (maxhaton) [reply std/datetime/systime.d:450]: I don't think there's any issue having a little redundancy here

Comment (Geod24) [reply std/datetime/systime.d:450]: I think it'd be better to link to core.time.hnsecs on the first use.

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**